source/parsing/Lexer.cpp

```diff
@@ -1239,7 +1239,10 @@ void Lexer::scanLineComment() {
             advance();
         }
         else {
-            sawUTF8Error |= !scanUTF8Char(sawUTF8Error);
+            auto error = !scanUTF8Char(sawUTF8Error);
+            sawUTF8Error |= error;
+            if (error)
```

Comment:
I don't think you need to do this; I would just remove the errorCount increment from scanUTF8Char, since the "error" it issues is actually a suppressible warning, and this is otherwise pretty clunky.

Reply:
As I wrote above, I did this to minimize the impact of this change. `scanUTF8Char` is also called by other routines and removing `errorCount++` will also affect error counting of other UTF8 errors, mainly on strings.
Assuming this is indeed your intention, I'll prepare another commit for this.

Reply:
Yeah, that was my intent. I think it's fine, the lexer error count thing is perhaps not the most impactful feature anyway.

Reply:
It's just rolling back the original code.